### PR TITLE
Fix up log messages for ProtoUtil.conflicts 

### PR DESCRIPTION
**Pull request by KentShikama**

### CODE REVIEW

**Comment by KentShikama** [comment]
bors try

**Comment by KentShikama** [comment]
bors try

File: casper/src/test/scala/coop/rchain/casper/HashSetCasperTest.scala [hunk 1]
```diff
@@ -73,8 +73,8 @@ class HashSetCasperTest extends FlatSpec with Matchers {
       deploy <- ProtoUtil.basicDeployData[Effect](0)
       _      <- MultiParentCasper[Effect].deploy(deploy)
 
-      _      = logEff.infos.size should be(1)
-      result = logEff.infos.head.contains("Received Deploy") should be(true)
+      _      = logEff.infos.size should be(2)
+      result = logEff.infos(1).contains("Received Deploy") should be(true)
```
Comment (ArturGajowy) [inline casper/src/test/scala/coop/rchain/casper/HashSetCasperTest.scala]: shouldn't `.head` be changed to `.infos(0)`?

Comment (KentShikama) [reply casper/src/test/scala/coop/rchain/casper/HashSetCasperTest.scala]: No, we now want to pick up the second element

**Pull request merged**